tools/roslaunch/src/roslaunch/substitution_args.py

```diff
@@ -190,8 +189,7 @@ def _find_resource(resolved, a, args, _context):
     path = _sanitize_path(args[1])
     # we try to find the specific path in share via catkin
     # which will search in install/devel space and the source folder of the package
-    # don't know why not imported at the top
-    from roslaunch.resource_cache import find_in_workspaces
+    from catkin.find_in_workspaces import find_in_workspaces
     paths = find_in_workspaces(['share'], project=args[0], path=path, first_matching_workspace_only=True, first_match_only=True)
```

Comment:
Since I don't have a depth camera attached I can't use that example. Also based on the size I doubt it will show a different performance for the `find_in_workspaces` calls.